packages/office-addin-project/src/convert.ts

```diff
@@ -37,9 +37,14 @@ export async function convertProject(
     return;
   }
   await backupProject(backupPath);
+  try {
+    await convert(manifestPath, outputPath, false, false);
+  } catch (err: any) {
+    console.log(`Error in conversion.`);
+    throw err;
+  }
```

Comment:
Is this all we need to do?  Isn't there any clean up from a partially completed call to convert.

Reply:
There would still be a `.zip` backup file that was created. Other than that. you shouldn't see other changes in the project (Just tested again and it is like that).

Reply:
Right now that may be the case in our limited experience, but there may be something we miss or something that get's changed that alters this.  The backup may be there, but we're leaving it up to the customer to switch back even when we detect something went wrong.  It is a better customer experience to report the error, but them back into the state the started with and the way to be sure of that is to restore the backup for them.